src/devices/Arduino/FirmataDevice.cs

```diff
@@ -775,7 +788,7 @@ private T PerformRetries<T>(int numberOfRetries, Func<T> operation)
             throw new TimeoutException("Timeout waiting for answer. Aborting. ", lastException);
         }
 
-        internal void SetPinMode(int pin, SupportedMode firmataMode)
+        internal void SetPinMode(int pin, byte firmataMode)
```

Comment:
I think this should take SupportedMode/Firmata mode instead, it's hard to reason about `byte`